lib/data/content/chapter-11.yml

```diff
@@ -42,24 +42,28 @@ case_studies:
+    text: "<blockquote>Protected and conserved areas have been impacted by the COVID-19 pandemic, but there is now a unique opportunity to support these areas through economic recovery plans that safeguard biodiversity, mitigate climate change, and reduce the risk of future pandemics</blockquote>
+    
+    The effects of COVID-19 on human health and economies are evident in protected areas and OECMs around the world. In addition to negative impacts on effectiveness, management capacity and budgets, communities living in and around these areas have experienced significant impacts on their livelihoods (Hockings <em>et al.</em>, 2020). \n          
           Nearly half of PA agencies surveyed across 19 African countries reported that basic operations were at risk in the short term from the restrictions imposed by COVID-19 (Waithaka, 2020). The loss of income resulting from these restrictions significantly affected their ability to pay salaries, monitor and enforce protection for wildlife and protect local communities from damage caused by wildlife. The sudden collapse of tourism revenues in protected areas has caused job losses, undermined development projects, and weakened collaboration with investors (Waithaka, 2020).\n         
-          Despite commitments around the world to “build back better”, the focus on short-term economic recovery risks putting pressure on environmental protections. One example relates to provisions authorising new or expanded extractive industries and infrastructure in protected and conserved areas (Conservation International, 2020). This is especially of concern in countries where these areas are already under severe pressure. However, protected and conserved areas have a central role in addressing ecosystem degradation and biodiversity loss, and therefore reducing the risk of future zoonotic pandemics in addition to providing other economic, social and environmental benefits.\n
+          Despite commitments around the world to “build back better”, the focus on short-term economic recovery risks putting pressure on environmental protections. One example relates to provisions authorising new or expanded extractive industries and infrastructure in protected and conserved areas  (Golden Kroner <span class='italic'>et al.,</span> 2021; Conservation International, 2020). This is especially of concern in countries where these areas are already under severe pressure. However, protected and conserved areas have a central role in addressing ecosystem degradation and biodiversity loss, and therefore reducing the risk of future zoonotic pandemics in addition to providing other economic, social and environmental benefits  (Ferreira <span class='italic'>et al.,</span> 2021).\n
```

Comment:
I notice that there are double spaces before the newly added references but it's not visible on the site